command/loadbot/loadbot_command.go

```diff
@@ -0,0 +1,213 @@
+		Description: "The specified gas limit",
+		Arguments: []string{
+			"LIMIT",
+		},
+		ArgumentsOptional: false,
+		FlagOptional:      true,
+	}
+
+	l.FlagMap["gasPrice"] = helper.FlagDescriptor{
+		Description: "The gas price",
+		Arguments: []string{
+			"GASPRICE",
+		},
+		ArgumentsOptional: false,
+		FlagOptional:      true,
+	}
+
+	l.FlagMap["url"] = helper.FlagDescriptor{
+		Description: "The URL used to make the transaction submission",
+		Arguments: []string{
+			"URL",
+		},
+		ArgumentsOptional: false,
+		FlagOptional:      false,
+	}
+
+	l.FlagMap["chain-id"] = helper.FlagDescriptor{
+		Description: "The Chain ID used to sign the transactions",
+		Arguments: []string{
+			"CHAIN_ID",
+		},
+		ArgumentsOptional: false,
+		FlagOptional:      true,
+	}
+
+	l.FlagMap["count"] = helper.FlagDescriptor{
+		Description: "The number of transactions to send",
+		Arguments: []string{
+			"COUNT",
+		},
+		ArgumentsOptional: false,
+		FlagOptional:      false,
+	}
+
+	l.FlagMap["value"] = helper.FlagDescriptor{
+		Description: "The value sent during the transaction by the loadbot",
+		Arguments: []string{
+			"VALUE",
+		},
+		ArgumentsOptional: false,
+		FlagOptional:      true,
+	}
+
+	l.FlagMap["grpc"] = helper.FlagDescriptor{
+		Description: "The gRPC url used by the loadbot to verify post load transactions status",
+		Arguments: []string{
+			"GRPC_URL",
+		},
+		ArgumentsOptional: false,
+		FlagOptional:      false,
+	}
+}
+
+func (l *LoadbotCommand) GetHelperText() string {
+	return "Runs the loadbot with the given properties (TPS, accounts...)"
+}
+
+func (l *LoadbotCommand) GetBaseCommand() string {
+	return "loadbot"
+}
+
+func (l *LoadbotCommand) Synopsis() string {
+	return l.GetHelperText()
+}
+
+func (l *LoadbotCommand) Help() string {
+	l.Meta.DefineFlags()
+	l.DefineFlags()
+
+	return helper.GenerateHelp(l.Synopsis(), helper.GenerateUsage(l.GetBaseCommand(), l.FlagMap), l.FlagMap)
+}
+
+func (l *LoadbotCommand) Run(args []string) int {
+	flags := flag.NewFlagSet(l.GetBaseCommand(), flag.ContinueOnError)
+
+	var tps uint64
+	var accountsRaw helperFlags.ArrayFlags
+	var gasLimit uint64
+	var gasPriceRaw string
+	var urls helperFlags.ArrayFlags
+	var chainID uint64
+	var count uint64
+	var valueRaw string
+	var gRPC string
+
+	flags.Uint64Var(&tps, "tps", 100, "")
+	flags.Var(&accountsRaw, "account", "")
+	flags.Uint64Var(&gasLimit, "gasLimit", 1000000, "")
+	flags.StringVar(&gasPriceRaw, "gasPrice", "0x100000", "")
+	flags.Var(&urls, "url", "")
+	flags.Uint64Var(&chainID, "chain-id", helper.DefaultChainID, "")
+	flags.Uint64Var(&count, "count", 1000, "")
+	flags.StringVar(&valueRaw, "value", "", "")
+	flags.StringVar(&gRPC, "grpc", "", "")
+
+	if err := flags.Parse(args); err != nil {
+		l.UI.Error(fmt.Sprintf("failed to parse args: %v", err))
+		return 1
+	}
+
+	// Parse accountsRaw
+	var addresses = []types.Address{}
+	if accountsRaw == nil {
+		l.UI.Error("failed to parse accountsRaw used by the loadbot")
```

Comment:
account is more proper name to show to user (accountsRaw is just variable name)

Reply:
I think its a typo due to refactor, thanks for mentioning it !